From felipe.balbi@nokia.com  Thu Apr 29 12:37:42 2010
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Thu, 25 Mar 2010 13:25:26 +0200
Subject: usb: musb: Makefile: remove unexistent config option
To: Greg KH <greg@kroah.com>
Message-ID: <1269516328-28267-10-git-send-email-felipe.balbi@nokia.com>


Remove the unexistent CONFIG_USB_INVENTRA_MUSB_HAS_AHB_ID
option from our Makefile.

Problem reported by Sergei Shtylyov <sshtylyov@mvista.com>

Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/Makefile |    6 ------
 1 file changed, 6 deletions(-)

--- a/drivers/usb/musb/Makefile
+++ b/drivers/usb/musb/Makefile
@@ -72,12 +72,6 @@ endif
 
 ################################################################################
 
-# FIXME remove all these extra "-DMUSB_* things, stick to CONFIG_*
-
-ifeq ($(CONFIG_USB_INVENTRA_MUSB_HAS_AHB_ID),y)
-	EXTRA_CFLAGS += -DMUSB_AHB_ID
-endif
-
 # Debugging
 
 ifeq ($(CONFIG_USB_MUSB_DEBUG),y)
